parastell/invessel_build.py

```diff
@@ -619,14 +620,15 @@ def poloidal_angles(self, angle_list):
             self._logger.error(e.args[0])
             raise e
 
-        self._poloidal_angles = angle_list
-        if self._poloidal_angles[-1] - self._poloidal_angles[0] > 360.0:
+        if angle_list[-1] - angle_list[0] != 360.0:
```

Comment:
Exactly equal in floating point math is tricky.

Should we check that these are close and then move one to make them exact if they are close, or make an error if they aren't close?

Reply:
Good point. I think as long as they're close (within a reasonably small tolerance) it's fine. We should raise the error if they're not close.